CDS/cdsresponder/cdsresponder.rb

```diff
@@ -87,10 +118,10 @@ def initialize(arn, routename, arg, notification, logging_conf)
   def GetUniqueFilename(path)
     filebase=@routename.gsub(/[^\w\d]/, "_")
     filename=path+'/'+filebase+".xml"
-    n=0;
-    while (Pathname(filename).exist?) do
+    n=0
+    while Pathname(filename).exist? do
```

Comment:
Just to make sure I understand what this does, it just makes sure that if there already is a file with that path, it will add a version or w.e to the end of the name is that right?